GUIDELINES.md

```diff
@@ -72,7 +72,7 @@ We try to interpret HTTP Status codes the right way:
  - `204` - No Content, on resource updates or actions
  - `400` - Bad Request, the request contains invalid data or references non-existing resources
  - `401` - Unauthorized, invalid or missing access token
- - `403` - Forbidden, not allowed to access this resource
+ - `403` - Forbidden, not allowed to access this resource, or a specific property of a resource
```

Comment:
We thought about returning a `400` when the User don't have permissions to access a specific property but it would be incorrect because the request is not malformed nor invalid, is indeed a matter of authorization.

Reply:
When reading this, this would also return a 403 when retrieving info about a resource with a property you don't have access to. Would change it to:

```suggestion
 - `403` - Forbidden, not allowed to access this resource or update a property you are not allowed to change
```